lxd/images.go

```diff
@@ -4467,8 +4477,9 @@ func imageExportPost(d *Daemon, r *http.Request) response.Response {
 
 	run := func(op *operations.Operation) error {
 		createArgs := &lxd.ImageCreateArgs{}
-		imageMetaPath := shared.VarPath("images", details.imageFingerprintPrefix)
-		imageRootfsPath := shared.VarPath("images", details.imageFingerprintPrefix+".rootfs")
+		imageMetaPath := s.ImagesStoragePath()
+		imageMetaPath = filepath.Join(imageMetaPath, details.imageFingerprintPrefix)
```

Comment:
I don't know if the compiler is smart enough to optimize away the first assignment and do just:

```suggestion
		imageMetaPath := filepath.Join(s.ImagesStoragePath(), details.imageFingerprintPrefix)
```
but I find this more compact/straight forward version easier to read. Probably a question of style/personal taste however :)

Reply:
Thanks for the review! I've changed the code as you suggest wherever there was only one reference to the new variable. If it's used on multiple places, I've left the code as it was.

Reply:
Thanks, you've struck the perfect balance IMHO ;)